mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeans.scala

```diff
@@ -103,21 +103,17 @@ class KMeans private (
   }
 
   /**
-   * :: Experimental ::
    * Number of runs of the algorithm to execute in parallel.
    */
   @Since("1.4.0")
-  @Experimental
```

Comment:
I want to keep this `Experimental` because I wonder most users don't need it but it complicates the implementation by a lot.